frog/imports/client/Wiki/index.js

```diff
@@ -303,22 +302,58 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     const parsedPages = wikiStore.parsedPages;
     const pageTitle = getPageTitle(parsedPages, this.state.initialPageTitle);
     const pageTitleLower = pageTitle.toLowerCase();
-    const fullPageObj = parsedPages[pageTitleLower];
+    let fullPageObj = parsedPages[pageTitleLower];
 
     if (!fullPageObj) {
       this.initialLoad = true;
       this.createPage(pageTitle, 3);
       return;
     }
 
-    const instanceId =
+    if (
+      fullPageObj.pageSettings !== undefined &&
+      fullPageObj.pageSettings.hidden &&
+      this.state.privilege !== PRIVILEGE_OWNER
+    ) {
+      this.props.showModal(
+        <AlertModal
+          title="Unable to view Page"
+          callback={() => {
+            this.props.hideModal();
+          }}
+        >
+          This page is hidden by the owner.
+        </AlertModal>
+      );
+      fullPageObj = parsedPages['home'];
```

Comment:
What happens if home is deleted?

Reply:
If users are able to delete home than we have a different problem.